tests/test_api_v1.py

```diff
@@ -153,6 +154,20 @@ def test_malformed_request_id(client, db):
     assert data == {'error': 'The requested resource was not found'}
 
 
+def test_create_request_invalid_flag(auth_env, client, db):
```

Comment:
It would be nice to add a valid test as well. (Will need to add test flags to the test db)

Reply:
Added a new test which submits a request when the flag is active and fetches the request when the flag is set to inactive. :)